bin/tapioca

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
```

Comment:
should we have documentation somewhere about what these things are and how they should be used during development

Reply:
Yeah that'd probably be a good idea - though currently this doesn't implement anything accessible to the SDK - just the WASM loading/init.